generated/niscope/enums.py

```diff
@@ -24,6 +24,117 @@ class AcquisitionType(Enum):
     '''
 
 
+class ArrayMeasurement(Enum):
```

Comment:
It's much less than ideal having these here and even more so having them in documentation.
How difficult would it be to add support for private enums?